projects/core/src/global-message/facade/global-message.service.ts

```diff
@@ -8,22 +8,21 @@ import {
   GlobalMessage,
   GlobalMessageType
 } from '../models/global-message.model';
+import { StateWithGlobalMessage, GlobalMessageEntities } from '../store/index';
```

Comment:
that import is redundant here, that is already in 'fromStore'

Reply:
thanks, will look into that. please create a bug ticket with a link to this conversation and assign it to me

Reply:
rebuilt imports